index.html

```diff
@@ -957,8 +957,20 @@ <h2><dfn>Coalesced events</dfn></h2>
             or <code>pointerrawupdate</code> event, the list is a sequence of all <code>PointerEvent</code>s
             that were coalesced into this event; otherwise it is an empty list.</p>
 
-            <p>The events in the coalesced event list will have increasing
-            <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a>
+            <p>Coalesced events MUST:</p>
+            <ul>
+                <li>have the same <code>pointerId</code>, <code>pointerType</code>, <code>isPrimary</code>,
+                    <a href="https://dom.spec.whatwg.org/#dom-event-istrusted"><code>isTrusted</code></a>,
+                    and <a href="https://dom.spec.whatwg.org/#event-target"><code>target</code></a> as
+                    the dispatched "parent" event</li>
```

Comment:
'parent' feels a bit weird. The old text tries to more programmatically initialize the events, that feels less error prone approach.

Reply:
kept the word `"parent"` in quotes, but expanded to make it clear what is meant (`dispatched "parent" pointer event that the <code>getCoalescedEvents()</code> method was called on`)